com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -144,8 +144,12 @@ public UniversalRenderPipeline(UniversalRenderPipelineAsset asset)
         {
             SetSupportedRenderingFeatures();
 
+            // In QualitySettings.antiAliasing disabled state uses value 0, where in URP 1
+            int qualitySettingsMsaaSampleCount = QualitySettings.antiAliasing > 0 ? QualitySettings.antiAliasing : 1;
+            bool msaaSampleCountNeedsUpdate = qualitySettingsMsaaSampleCount != asset.msaaSampleCount;
+
             // Let engine know we have MSAA on for cases where we support MSAA backbuffer
-            if (QualitySettings.antiAliasing != asset.msaaSampleCount)
+            if (msaaSampleCountNeedsUpdate)
             {
                 QualitySettings.antiAliasing = asset.msaaSampleCount;
```

Comment:
don't we need to convert the data when setting antialiasing as well?

Reply:
@thomas-zeng I see @lukaschod already checked SetMSAALevel but pinging you here for awareness

Reply:
conversion happens inside native side